composer.json

```diff
@@ -12,29 +13,35 @@
   },
   "require-dev": {
     "codeception/codeception": "^2.0.0",
-    "phpstan/phpstan": "^0.9.2 || ^0.10.0"
-  },
-  "conflict":  {
-    "spryker/new-relic": "*",
-    "spryker/new-relic-api": "*"
+    "phpstan/phpstan": "^0.9.2 || ^0.10.0",
```

Comment:
we might also want to switch to 0.12 here at some point :)
Can also be a follow up